### ISSUE TITLE
Simplify Mixins

### ISSUE THREAD

**Issue by jamiebuilds** [curvedmark/roole#44]
``` sass
@mixin name($option)
    property: $option

.selector
    @name(value)
```

The @ symbol could be replaced with anything, although I like how it follows CSS's default usage of the @ symbol... and people could use sass syntax highlighters before their editor gains Roole support.

Also if Issue #15 for built in function support goes in I think it should follow this same way of writing:

``` sass
@function name($option)
    @return $option

.selector
    property: name(value)
```

**Comment by curvedmark** [comment]
I want to give javascript developers a sense of familiarity in that functions(or mixins) are first-class citizens. #17 even tries to pass mixins directly as arguments, but I haven't found a solid use case for it.

**Comment by jamiebuilds** [comment]
What about giving CSS/SASS developers a sense of familiarity? 

And still, this is is even closer to javascript:

``` scss
// Javascript:
function name(arguments) {
    return something;
}
name(arguments);

// Roole
@function name(arguments)
    @return something

selector
    property: name(arguments)
```

They are being called and set the same way (only without brackets and with the @ symbol to define constructors). Sure you can put functions inside of variables in javascript, but it's not necessary.

In this case it would only make the code longer. I'm not even sure it should be the full word function (after all people complain about how long of a word it is in javascript all the time, which is why a new => syntax is being added).

**Comment by curvedmark** [comment]
This syntax does not make it obvious that functions are just regular values. Also two separate date types is unnecessary. Functions and mixins can be defined in the same way and only differs on how they are called.

**Comment by jamiebuilds** [comment]
I think that developers are very aware of how functions work. As for mixins I think they should follow the same syntax:

``` sass
@mixin one(arguments)
@function two(arguments)


selector
    @one(arguments)
    property: two(arguments)
```

Why would you make it any more lengthy, when it's clear with SASS's widespread adoption of this syntax is so popular? 

I like how SASS does function calling because it makes things easier to read, I write stuff like `rgba(black, .2)` all the time and it fits really well inside of writing things like gradients.

For mixins I think Stylus does an excellent job:

``` sass
selector
    mixin-name arguments
    property value
```

**Comment by curvedmark** [comment]
Landed in 76eb2ea6f9677a4ea51df6db7ee5b340c31fa194

The implemented syntax is more consistent with constructs like `@media`, `@if`, `@for` that no quotes are needed for parameters.
